downloads.md

```diff
@@ -62,7 +62,7 @@ where `{distribution}` is your distribution name, wokr only for one of the list:
 
 ### RPM
 
-There is no possibility for the moment with development PRM packages
+There is no possibility for the moment to have RPM packages of development version
```

Comment:
It is not possible currently to have the develop branch version with RPM packages.